tobspr-games/shapez-community-edition
# Refactor/yarn

## This pull request updates yarn 1 to 4 and also some other stuff

### Changes
* Updated from yarn 1 to yarn 4
* Removed FTP task (mostly because it was interfering with the package install)
* Removed "shapez.io-private-artifacts" from electron package.json
* Changed README.md accordingly

### How to test
1. Update node to [20.11](https://nodejs.org/en/download/)
2. `corepack enable`
3. yarn 4 should be installed
4. Just follow the build steps as usual
